datalad_dataverse/tests/test_create_sibling_dataverse.py

```diff
@@ -0,0 +1,33 @@
+from requests.exceptions import ConnectionError
+from datalad.tests.utils_pytest import (
+    assert_raises,
+    assert_result_count,
+    skip_if,
+    with_tempfile,
+)
+from datalad.api import (
+    create_sibling_dataverse,
+)
+from datalad.distribution.dataset import (
+    Dataset,
+)
+from datalad.support.exceptions import IncompleteResultsError
+
+from datalad_dataverse.tests import (
+    API_TOKENS,
+    DATAVERSE_URL
+)
+
+
+@skip_if(cond=not DATAVERSE_URL)
+@with_tempfile
+def test_dummy(path=None):
```

Comment:
I'd say we skip the test until there is an actual implementation, and then merge this PR. OK @bpoldrack ?

Reply:
Ok. Just wanted to show how to use that decorator